docs/pages/docs/guides/custom-admin-ui-navigation.md

```diff
@@ -323,6 +375,29 @@ type NavItemProps = {
 By default the `isSelected` value will be evaluated by the condition `router.pathname === href`.
 Pass in `isSelected` if you have a custom condition or would like more granular control over the "selected" state of Navigation items.
 
+### NavItemGroup
+
+This component is a styled unordered list `<ul>` which should be used to wrap `NavItem`, `ListNavItem` and `comp
+
+```tsx
+import { NavItemGroup } from '@keystone-6/core/admin-ui/components'
+```
+
+{% hint kind="warn" %}
+Versions of `@keystone-6/core` before `1.2.0` wrapped all children of `NavigationContainer` with the `NavItemGroup` component.
```

Comment:
I've guessed what the version number will be. Also I'm not sure we need to call out that this thing has changed.